mne/preprocessing/ecg.py

```diff
@@ -291,6 +293,18 @@ def create_ecg_epochs(raw, ch_name=None, event_id=999, picks=None, tmin=-0.5,
 
         .. versionadded:: 0.14.0
 
+    qrs_threshold : float | str
+        Between 0 and 1. qrs detection threshold. Can also be "auto" to
+        automatically choose the threshold that generates a reasonable
+        number of heartbeats (40-160 beats / min).
+
+    filter_length : str | int | None
+        Number of taps to use for filtering.
+
+    tstart : float
+        Start detection after tstart seconds. Useful when beginning
+        of run is noisy.
```

Comment:
I don't think we need this one, can just use `raw.crop`. It was probably in `find_ecg_events` before we had this